tests/test_pipelines_automatic_speech_recognition.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from transformers import AutoFeatureExtractor, AutoTokenizer, Speech2TextForConditionalGeneration, Wav2Vec2ForCTC
+from transformers.pipelines import AutomaticSpeechRecognitionPipeline
+from transformers.testing_utils import require_torch, require_torchaudio, slow
+
+
+# from .test_pipelines_common import CustomInputPipelineCommonMixin
+
+
+class AutomaticSpeechRecognitionPipelineTests(unittest.TestCase):
```

Comment:
we should add a `require_datasets` here I think

Reply:
Done !